kubernetes/deployments/hello.yaml

```diff
@@ -1,9 +1,13 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: hello
 spec:
   replicas: 3
+  selector:
+    matchLabels:
+    app: hello
```

Comment:
Typo:

```
selector:
  matchLabels:
    app: hello
    track: stable
```

Reply:
hello i have it below this comment .. can u confirm the same ?